assets/css/default.css

```diff
@@ -1,3 +1,43 @@
+.channel-owner {
+  background-color: #008BEC;
+  color: beige;
```

Comment:
I would try and use colors already present in this file, although I wouldn't consider it a deal-breaker and I'll still be happy to merge. Here, you might consider using `#fff`.